codec-http2/src/test/java/io/netty/handler/codec/http2/WeightedFairQueueByteDistributorDependencyTreeTest.java

```diff
@@ -584,8 +584,8 @@ public void sameNodeDependentShouldNotStackOverflowNorChangePrioritizableForTree
         setPriority(streamC.id(), streamA.id(), DEFAULT_PRIORITY_WEIGHT, false);
         setPriority(streamD.id(), streamA.id(), DEFAULT_PRIORITY_WEIGHT, true);
 
-        boolean[] exclusives = new boolean[] {true, false};
-        short[] weights = new short[] { DEFAULT_PRIORITY_WEIGHT/*, 100, 200, DEFAULT_PRIORITY_WEIGHT */};
+        boolean[] exclusives = {true, false};
+        short[] weights = { DEFAULT_PRIORITY_WEIGHT/*, 100, 200, DEFAULT_PRIORITY_WEIGHT */};
```

Comment:
also does our style guide care about spaces after the `{`? the line before has no spaces and this one has a space.